sdcm/cluster_aws.py

```diff
@@ -121,6 +121,14 @@ def __str__(self):
     def calculate_spot_duration_for_test(self):
         return floor(self.test_config.TEST_DURATION / 60) * 60 + 60
 
+    @property
+    def instance_profile_name(self) -> str | None:
+        if self.node_type == "scylla-db":
+            return self.params.get('aws_instance_profile_name_db')
+        if self.node_type == "loader":
+            return self.params.get('aws_instance_profile_name_loader')
+        return None
```

Comment:
what about monitor?
Shouldn't be there some default for other cases?

Reply:
we don't have a use case yet that we need it for monitor.

when we will, we'll add this configuration and code to handle it.